guidelines/application_requirements.md

```diff
@@ -0,0 +1,84 @@
+# Guidelines for Application Requirements
+
+Authors: chasm@google.com, jlewi@google.com, kam.d.kasravi@intel.com
+
+## Objective
+
+Define requirements and recommendations aimed at graduating applications (e.g. controller and web apps) in Kubeflow to 1.0.
+
+## Scope
+
+This document is intended to cover applications that are deployed in the Kubernetes cluster and in particular:
+
```

Comment:
nit: ... in which Kubernetes cluster? The cluster that also contains `kubeflow`?